packages/cli/package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "zapier-platform-cli",
+  "version": "12.2.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "yarn": {
+      "version": "1.22.19",
+      "resolved": "https://registry.npmjs.org/yarn/-/yarn-1.22.19.tgz",
+      "integrity": "sha512-/0V5q0WbslqnwP91tirOvldvYISzaqhClxzyUKXYxs07yUILIs5jx/k6CFe8bvKSkds5w+eiOqta39Wk3WxdcQ=="
+    }
+  }
+}
```

Comment:
`package-lock.json` should not be in this change. We should undo the change here.